packages/geo/src/lib/filter/ogc-filterable-item/ogc-filterable-item.component.html

```diff
@@ -6,29 +6,13 @@
   <h4 *ngIf="ogcFiltersHeaderShown" matLine [matTooltip]="layer.title" matTooltipShowDelay="500">{{layer.title}}</h4>
   <h4 *ngIf="!ogcFiltersHeaderShown" matLine></h4>
 
-  <button mat-icon-button *ngIf="hasActiveSpatialFilter && showFeatureOnMap" tooltip-position="below" matTooltipShowDelay="500"
-  [matTooltip]="'igo.geo.filter.hideFeatureExtent' | translate"
-  [disabled]="!layer.visible || !layer.isInResolutionsRange"
-  [color]="color"
-  (click)="toggleShowFeatureOnMap()">
-  <mat-icon svgIcon="border-outer"></mat-icon>
-  </button>
-
-  <button mat-icon-button *ngIf="hasActiveSpatialFilter && !showFeatureOnMap" tooltip-position="below" matTooltipShowDelay="500"
-  [matTooltip]="'igo.geo.filter.showFeatureExtent' | translate"
-  [disabled]="!layer.visible || !layer.isInResolutionsRange"
-  [color]="color"
-  (click)="toggleShowFeatureOnMap()">
-    <mat-icon svgIcon="border-none"></mat-icon>
-  </button>
-
   <span *ngIf="downloadable && ogcFiltersHeaderShown">
     <button mat-icon-button tooltip-position="below" matTooltipShowDelay="500" [matTooltip]="'igo.geo.download.action' | translate"
       [color]="color" (click)="openDownload()">
       <mat-icon svgIcon="download"></mat-icon>
     </button>
   </span>
-  <button [disabled]="!datasource.options.sourceFields || datasource.options.sourceFields.length === 0" mat-icon-button tooltip-position="below" matTooltipShowDelay="500"
+  <button *ngIf="datasource.options.ogcFilters.advancedOgcFilters" [disabled]="!datasource.options.sourceFields || datasource.options.sourceFields.length === 0" mat-icon-button tooltip-position="below" matTooltipShowDelay="500"
```

Comment:
Line too long. Maybe you could move ngIf and disabled logics into methods.

Reply:
Done